test/fs/test_fs_js_api.c

```diff
@@ -37,6 +37,39 @@ int main() {
 #endif
     );
 
+    /********** test FS.rename() **********/
+    EM_ASM(
+        FS.mkdir('renamedir');
+        FS.writeFile('renamedir/renametestfile', "");
+        FS.writeFile('toplevelfile', "");
```

Comment:
This line could be moved to later down, before the try-catch in which we test it having an effect.